From 661e46685569eaaf5eae0594b4473b073bea2738 Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Tue, 26 Nov 2013 16:14:50 +0800
Subject: [PATCH] bcm-xlp: update ETHTOOL_MAC_SEC_TX/RX values

The commands ETHTOOL_MAC_SEC_TX/RX[_MEM] are XLP special, use the proper
values to avoid the potential conflicts with the values of common commands
in the future.

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 include/uapi/linux/ethtool.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/uapi/linux/ethtool.h b/include/uapi/linux/ethtool.h
index 8dbc7e2..f607959 100644
--- a/include/uapi/linux/ethtool.h
+++ b/include/uapi/linux/ethtool.h
@@ -916,10 +916,10 @@ enum ethtool_sfeatures_retval_bits {
 #define ETHTOOL_SEEE		0x00000045 /* Set EEE settings */
 
 #ifdef CONFIG_NLM_XLP
-#define	ETHTOOL_MAC_SEC_TX	0x00000046 /* MACSec TX config to device */
-#define	ETHTOOL_MAC_SEC_TX_MEM	0x00000047 /* MACSec TX mem config to device */
-#define	ETHTOOL_MAC_SEC_RX	0x00000048 /* MACSec RX config to device */
-#define	ETHTOOL_MAC_SEC_RX_MEM	0x00000049 /* MACSec RX mem config to device */
+#define ETHTOOL_MAC_SEC_TX      0x0000F001 /* MACSec TX config to device */
+#define ETHTOOL_MAC_SEC_TX_MEM  0x0000F002 /* MACSec TX mem config to device */
+#define ETHTOOL_MAC_SEC_RX      0x0000F003 /* MACSec RX config to device */
+#define ETHTOOL_MAC_SEC_RX_MEM  0x0000F004 /* MACSec RX mem config to device */
 #endif
 
 /* compatibility with older code */
-- 
1.8.4.93.g57e4c17

